studio/components/interfaces/Auth/AuthProvidersForm/FormField.tsx

```diff
@@ -28,35 +28,35 @@ const FormField: FC<Props> = ({ name, properties, formValues, disabled = false }
           type={hidden ? 'password' : 'text'}
           label={properties.title}
           labelOptional={
-            properties.descriptionOptional ? (
```

Comment:
Someone not got Prettier installer ?

Reply:
Apoloiges, yeah autolinting was borked on my editor. Have fixed.